lib/views/widgets/navigation/bottom_navbar.dart

```diff
@@ -21,7 +21,7 @@ class BottomNavBar extends HookConsumerWidget {
         NotificationsRouter(),
         SettingsRouter(),
         SampleCrud(),
-        AddTransactionRouter(),
+        // AddTransactionRouter(),
```

Comment:
If we don't need this, delete it totally.
or put comment why it is commented out

Reply:
ah, sorry i forgot to remove it miss. thank you 🙇‍♀️